email_templates/mentor_confirmation/text

```diff
@@ -0,0 +1,17 @@
+Hey {first_name},
```

Comment:
This should be "Hi" instead of "Hey" as per my email. Sorry for not clarifying which things need to be changed.